homeassistant/components/hive.py

```diff
@@ -0,0 +1,707 @@
+"""Hive Integration - Platform."""
+import logging
+from datetime import datetime
+from homeassistant.helpers.discovery import load_platform
+
+REQUIREMENTS = ['pyhiveapi==0.1.1']
+
+HGO = None
+_LOGGER = logging.getLogger(__name__)
+DOMAIN = 'hive'
+

```

Comment:
[Configuration validation](https://home-assistant.io/developers/code_review_component/#configuration) is missing.

Reply:
configuration validation added